AsyncDisplayKit/Details/_ASDisplayView.mm

```diff
@@ -375,3 +378,126 @@ - (UIView *)preferredFocusedView
 }
 #endif
 @end
+
+
+#pragma mark - Accessibility
+
+static const char *ASDisplayNodeAssociatedNodeKey = "ASAssociatedNode";
+
+@implementation UIAccessibilityElement (_ASDisplayView)
+
+- (void)setAsyncdisplaykit_node:(ASDisplayNode *)node
+{
+  objc_setAssociatedObject(self, ASDisplayNodeAssociatedNodeKey, node, OBJC_ASSOCIATION_ASSIGN); // Weak reference to avoid cycle, since the node retains the layer.
```

Comment:
We do not deal with UIView's if the node is layer backed or for the subnodes if shouldRasterizeSubnodes is true and to be able to convert the frame from a UIAccessibilityElement to screen coordinates in accessibilityElementAtIndex:. Please see also comments in accessibilityElementAtIndex:.